pkg/api/k8s_versions.go

```diff
@@ -182,16 +188,22 @@ func getKubeConfigs() map[string]map[string]string {
 
 func getVersionOverrides(v string) map[string]string {
 	switch v {
+	case "1.18.6":
```

Comment:
Updated pkg\api\common\versions.go to set these supported versions to true.

Reply:
have a full sync from aks-rp https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/3469550?_a=files&path=%2Fcommon%2Fk8sversions%2Fversions.go

Reply:
done, thanks.